roles/prune_catalog/tasks/main.yml

```diff
@@ -126,11 +126,14 @@
           -t {{ pc_destination_catalog }}
           -f {{ pc_tmp_dir }}/configs.Dockerfile .
       register: build_result
-      changed_when: build_result.rc != 0
-      notify:
-        - "Delete temp directory"
```

Comment:
I think it was not called because the `changed_when` condition was never met unless failed, now that the `changed_when` is set for success, it should trigger the notification, or did you face any issue that way?

Reply:
Yes, that was the issue. But IMO it is better not with the block

Reply:
The previous job failure is not related to this change.